crates/trie/sparse/src/trie.rs

```diff
@@ -933,7 +976,8 @@ where
                             if let Some(node) = self.provider.blinded_node(child_path.clone())? {
                                 let decoded = TrieNode::decode(&mut &node[..])?;
                                 trace!(target: "trie::sparse", ?child_path, ?decoded, "Revealing remaining blinded branch child");
-                                self.reveal_node(child_path.clone(), decoded)?;
+                                // TODO: fetch hash mask
```

Comment:
todo?

Reply:
plan to address it in the next PR, as it requires modifying the `BlindedProvider` trait

Reply:
replaced with a comment